app/konnect/api/filtering.md

```diff
@@ -0,0 +1,167 @@
+---
+title: Filtering
+content-type: reference
+---
+
+
+
+The Konnect API supports the ability to filter over the collection and
+only return results that you are interested in.
+This reference document explains how filtering works in {{site.konnect_short_name}} APIs.
+
+### Available fields
+
+* **Users**: `id`, `email`, `full_name`, `active`
+* **Teams**: `id`, `name`, `system_team`
+* **Assigned Roles**: `role`,`entity_type`, `entity_region`, `entity_id`
+
+### Numeric and timestamp fields
+
+When matching against both numeric and timestamp fields, you can filter in the following formats:
+
+* **Equal**: `?filter[field][eq]=value` or `?filter[field]=value`
+* **Contains**: `?filter[field][contains]=value`
+* **Less Than**: `?filter[field][lt]=value`
+* **Less Than or Equal**: `?filter[field][lte]=value`
+* **Greater Than**: `?filter[field][gt]=value`
+* **Greater Than or Equal**: `?filter[field][gte]=value`
+
+If the qualifier is omitted, `?filter[field]=value` for example, the filtering behavior will perform an **exact** style
+equal match.
+
+## Examples
+
+Given a response schema of:
+
+```json
+{
+  "data": [
+    {
+      "id": "500d74f4-37e1-4f59-b51a-8cf7c7903692",
+      "email": "Charlie@konghq.com",
+      "name": "Charlie",
+      "full_name": "Charlie Cruz",
+      "active": true,
+      "created_at": "2022-05-10T15:10:25Z"
+    },
+    {
+      "id": "500d74f4-37e1-4f59-b51a-8cf7c7903692",
+      "email": "Alex@konghq.com",
+      "name": "Alex",
+      "full_name": "Alex Cruz",
+      "active": true,
+      "created_at": "2022-05-10T15:10:25Z",
+      "updated_at": "2022-10-19T15:33:02Z"
+    }
+  ]
+}
+```
+
+### Single Filter
```

Comment:
suggestion: A single filter can still return multiple results. It might be educational to include the `filter[name][contains]=Cruz` example to show that >1 result can be returned. That would be most useful if the whole collection includes a non-matching record, like "Alex Garcia".